components/bl-rating-component/src/rating-icon.js

```diff
@@ -0,0 +1,69 @@
+import { icons } from './icons.js';
+
+const halfIconStyle = { width: `50%`, overflow: 'hidden', position: 'absolute' };
```

Comment:
could we move it to less, i think if we add one more class for half icon that will be possible